databricks/sdk/config.py

```diff
@@ -473,3 +498,36 @@ def deep_copy(self):
         """Creates a deep copy of the config object.
         """
         return copy.deepcopy(self)
+
+    def get_compute(self):
+        return Compute(self)
+
+
+class Compute:
```

Comment:
Is this class really necessary? Is there a downside to attaching the `is_serverless()` and `get_cluster_id()` methods to `Config` directly? Then all of lines 501-533 become
```python
  @property
  def is_serverless(self):
    return self.compute == "serverless"

  @property 
  def get_cluster_id(self):
    if self.compute.startswith("cluster:"):
      return self.compute[len("cluster:"):]
    return self.cluster_id
```